packages/builder/test/builtin-tasks/migrate.ts

```diff
@@ -0,0 +1,73 @@
+import { assert } from "chai";
+import * as fsExtra from "fs-extra";
+import fs from "fs";
+
+import { ERRORS } from "../../src/internal/core/errors-list";
+import { useEnvironmentWithBeforeEach } from "../helpers/environment";
+import { expectBuilderErrorAsync } from "../helpers/errors";
+import { useFixtureProject } from "../helpers/project";
+import { TASK_CLEAN, TASK_MIGRATE } from "../../src/builtin-tasks/task-names";
+import { AlgobRuntimeEnv } from "../../src/types";
+
+const outputFile = "output.txt"
+
+describe("migration task", function () {
+  useFixtureProject("scripts-dir");
+  useEnvironmentWithBeforeEach(async (algobEnv: AlgobRuntimeEnv) => {
+    await algobEnv.run(TASK_CLEAN, {});
```

Comment:
async - await is not needed here.

Reply:
It is needed because `run` is async already. And I want to execute only after `clean` finishes. 
What would be your way of doing it?

Reply:
Async is a syntactic sugar in JS.
Use this and you will see that it works:
```
  useEnvironmentWithBeforeEach((algobEnv: AlgobRuntimeEnv) => {
    return algobEnv.run(TASK_CLEAN, {});})
```

- [ ] remove async/await when it's not needed.